### DISCUSSION TITLE
Didn't reset!

### DISCUSSION THREAD

**Discussion by mohit0129** [community/community#62638] [Codespaces] [answered]
I've used codespaces service in july 31st lastly, but then when i open codespaces in August, it still shows that you've used 100%, even though it's new month, didn't reset! Can anybody help me with this? It's a free plan though!

**Comment by nathan-hello** [comment, marked as answer]
https://github.com/settings/billing

Right above where it says your usage for Codespaces, it says when it resets.

**Reply by mohit0129** [reply]
Oh okay ! Thank you!
